meshtastic/interfaces/ble/interface.py

```
@@ -3292,12 +2963,12 @@ def close(self) -> None:
                 # persistent discovery client/resources before downstream
                 # shutdown steps. This does not guarantee immediate cancellation
                 # of all in-flight scan/connect operations.
-                self.error_handler._safe_cleanup(
+                self.error_handler.safe_cleanup(
                     discovery_manager.close, "discovery manager close"
                 )
 
             self._set_receive_wanted(False)  # Tell the thread we want it to stop
```

Ruff (0.15.5)

[warning] Boolean positional value in function call

(FBT003)